pkg/controller/perconaservermongodb/version.go

```diff
@@ -264,7 +264,7 @@ func (r *ReconcilePerconaServerMongoDB) getVersionMeta(ctx context.Context, cr *
 		BackupVersion:          cr.Status.BackupVersion,
 		BackupsEnabled:         cr.Spec.Backup.Enabled && len(cr.Spec.Backup.Storages) > 0,
 		ShardingEnabled:        cr.Spec.Sharding.Enabled,
-		ClusterWideEnabled:     len(watchNs) == 0,
+		ClusterWideEnabled:     watchNs == "" || strings.Contains(watchNs, ","),
```

Comment:
A watchNs= "somenamespace," will be valid for this case? Maybe it is better to split the string and count that we have >1 splits which will mean that we have 2 or more namespaces 

e.g. watchNs= "somenamespace,someothernamespace" will create 2 splits, split1: "somenamespace" and split2: "someothernamespace"

I think we can also include this implementation in the unit tests. We can perform a very small refactor to this part of the unit test we already (`Test_majorUpgradeRequested`)  have set the `currentNs` value from the test parameters instead of having it hardcoded.